freqtrade/optimize/backtesting.py

```diff
@@ -207,6 +207,11 @@ def load_bt_data(self) -> Tuple[Dict[str, DataFrame], TimeRange]:
         """
         self.progress.init_step(BacktestState.DATALOAD, 1)
 
+        if self.config.get('freqai') is not None:
+            self.required_startup += int(self.config.get('freqai', {}).get('startup_candles', 1000))
```

Comment:
Wouldn't defaulting to 0 here make more sense?

if it's not set - we should assume that the value is 0 - not 1000 ...